src/ngRoute/directive/ngView.js

```diff
@@ -26,6 +26,12 @@ ngRouteModule.directive('ngView', ngViewFillContentFactory);
  *
  * The enter and leave animation occur concurrently.
  *
+ * @knownIssue If `ngView` is contained in an asynchronously loaded templated (i.e. not directly
```

Comment:
template~~d~~
And I would list the usual use-cases instead  (e.g. in an ngInclude or in a directive's templateUrl)